grid-proxy/internal/explorer/converters.go

```diff
@@ -70,7 +80,7 @@ func nodeFromDBNode(info db.Node) types.Node {
 		RentedByTwinID:    uint(info.RentedByTwinID),
 		SerialNumber:      info.SerialNumber,
 		Power:             types.NodePower(info.Power),
-		HasGPU:            info.HasGPU,
+		NumGPUs:           getNumGPUs(info.HasGPU),
```

Comment:
should be NumGPU not NumGPUs similar to runtime.NumCPU https://pkg.go.dev/runtime#NumCPU

Reply:
the field will be `num_gpus` right? I'd rather keep them consistent

Reply:
I agree, let's stay indeed consistent with the field in tfchain, i updated the title to make it `num_gpu` 
https://github.com/threefoldtech/tfchain/issues/756